studio/styles/main.scss

```diff
@@ -211,3 +211,7 @@ div.fixed.inset-0 {
     @apply text-sm;
   }
 }
+
+.custom-input-width div div div input {
+  width: 80%;
```

Comment:
we shouldn't be doing a width percentage fix like this - on larger screens it'll look awkward
action props for Input are typically of fixed width size, so a padding fix will do the trick

ideally this should be fixed in the UI library though since it affects any Input with any action prop. i'll do up a separate fix for now just for this one nonetheless